install/00_namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-cluster-network-operator
+  labels:
+    name: openshift-cluster-network-operator
```

Comment:
https://github.com/openshift/machine-config-operator/blob/3a73b1d142eb1efb3fad5b160915e6ddf3d38311/install/00_namespace.yaml#L7
`openshift.io/run-level: "1"` might be needed as `openshift-apiserver` might not be up.

Reply:
definitely, good catch